<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AStylePreferences</class>
 <widget class="QWidget" name="AStylePreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>648</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabIndentqtion">
      <attribute name="title">
       <string comment="@title:tab">Indentation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="KComboBox" name="cbIndentType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string comment="@item:inlistbox Indentation type">Tabs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item:inlistbox Indentation type">Force tabs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item:inlistbox Indentation type">Spaces</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="spacer13_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="inpNuberSpaces">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string comment="@info:tooltip">Number of spaces that will be converted to a tab.
The number of spaces per tab is controlled by the editor.</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="chkConvertTabs">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string comment="@info:tooltip">Convert tabs to spaces.</string>
         </property>
         <property name="text">
          <string comment="@option:check">Con&amp;vert tabs into spaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkFillEmptyLines">
         <property name="toolTip">
          <string comment="@info:tooltip">Fill empty lines with the white space of their previous lines.</string>
         </property>
         <property name="text">
          <string comment="@option:check">Fill empt&amp;y lines</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblIndentObjects">
           <property name="text">
            <string comment="@label:listbox">Indent:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="2">
          <widget class="QListWidget" name="listIdentObjects">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string comment="@item:inlistbox">Blocks</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item:inlistbox">Brackets</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item:inlistbox">Cases</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item:inlistbox">Class</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item:inlistbox">Labels</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item:inlistbox">Namespaces</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item:inlistbox">Preprocessor directives</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item:inlistbox">Switches</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QSpinBox" name="inpMaxStatement">
           <property name="toolTip">
            <string comment="@info:tooltip">Indent a maximal # spaces in a continuous statement,
relative to the previous line.</string>
           </property>
           <property name="value">
            <number>40</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="TextLabel3_2">
           <property name="toolTip">
            <string comment="@info:tooltip">Indent a minimal # spaces in a continuous conditional
belonging to a conditional header.</string>
           </property>
           <property name="text">
            <string comment="@label:spinbox">Minimum in conditional:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="inpMinConditional">
           <property name="toolTip">
            <string comment="@info:tooltip">Indent a minimal # spaces in a continuous conditional
belonging to a conditional header.</string>
           </property>
           <property name="specialValueText">
            <string comment="@item">Twice current</string>
           </property>
           <property name="minimum">
            <number>-1</number>
           </property>
           <property name="value">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="TextLabel2_2">
           <property name="toolTip">
            <string comment="@info:tooltip">Indent a maximal # spaces in a continuous statement,
relative to the previous line.</string>
           </property>
           <property name="text">
            <string comment="@label:spinbox">Maximum in statement:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="inpContinuation">
           <property name="toolTip">
            <string comment="@info:tooltip">Number of indents</string>
           </property>
           <property name="maximum">
            <number>4</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="chkAfterParens">
           <property name="toolTip">
            <string comment="@info:tooltip">Indent, instead of align, continuation lines following lines that contain an opening paren '(' or an assignment '='.</string>
           </property>
           <property name="text">
            <string comment="@option:check">Continuation lines:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>144</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOther">
      <attribute name="title">
       <string comment="@title:tab">Other</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="gpBrackets">
         <property name="title">
          <string comment="@title:group">Brackets</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="lblBrackets">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string comment="@label:listbox">Type:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QCheckBox" name="chkBracketsCloseHeaders">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string comment="@info:tooltip">Break brackets before closing headers (e.g. 'else', 'catch', ...)
from their immediately preceding closing brackets.</string>
              </property>
              <property name="text">
               <string comment="@option:check">Brea&amp;k closing headers</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="KComboBox" name="cbBrackets">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string comment="@item:inlistbox brackets style">No change</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="@item:inlistbox brackets style">Attach</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="@item:inlistbox brackets style">Break</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="@item:inlistbox brackets style">Linux</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="@item:inlistbox brackets style">Run-In</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QCheckBox" name="chkBracesAdd">
              <property name="toolTip">
               <string comment="@info:tooltip">Add braces to unbraced one-line conditional statements (e.g. 'if', 'for', 'while')</string>
              </property>
              <property name="text">
               <string comment="@option:check add braces to unbraced one-line conditional statements">Add b&amp;races</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gpBlocks">
         <property name="title">
          <string comment="@title:group">Blocks</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="chkBlockBreak">
            <property name="toolTip">
             <string comment="@info:tooltip">Insert empty lines around unrelated blocks, labels, classes,...
Known problems:

1. If a statement is NOT part of a block, 
the following statements are all double spaced. 
Statements enclosed in a block are formatted 
correctly.

2. Comments are broken from the block.
</string>
            </property>
            <property name="text">
             <string comment="@option:check break unrelated blocks by a newline">&amp;Break blocks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkBlockBreakAll">
            <property name="toolTip">
             <string comment="@info:tooltip">Like --break-blocks, except also insert empty lines 
around closing headers (e.g. 'else', 'catch', ...).

Known problems:

1. If a statement is NOT part of a block, 
the following statements are all double spaced. 
Statements enclosed in a block are formatted 
correctly.

2. Comments are broken from the block.
</string>
            </property>
            <property name="text">
             <string comment="@option:check break all blocks with a newline">Break all bl&amp;ocks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkBlockIfElse">
            <property name="toolTip">
             <string comment="@info:tooltip">Break 'else if()' statements into two different lines.</string>
            </property>
            <property name="text">
             <string comment="@option:check break else if() into separate lines">Break i&amp;f-else</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gpPadding">
         <property name="title">
          <string comment="@title:group">Padding</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="1">
             <widget class="KComboBox" name="cbParenthesisPadding">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string comment="@item:inlistbox parenthesis padding">No change</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="@item:inlistbox parenthesis padding">Unpad</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="@item:inlistbox parenthesis padding">Inside only</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="@item:inlistbox parenthesis padding">Outside only</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="@item:inlistbox parenthesis padding">Inside and outside</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QCheckBox" name="chkPadOperators">
              <property name="toolTip">
               <string comment="@info:tooltip">Insert space padding around operators.
Once padded, operators stay padded.
There is no unpad operator option.</string>
              </property>
              <property name="text">
               <string comment="@option:check">&amp;Add spaces around operators</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lblParenthesis">
              <property name="text">
               <string comment="@label:listbox">Pad parenthesis:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QCheckBox" name="chkPadParenthesisHeader">
              <property name="toolTip">
               <string comment="@info:tooltip">Insert space padding after parenthesis headers (e.g. 'if', 'for', 'while', ...)</string>
              </property>
              <property name="text">
               <string comment="@option:check">Add spaces after parenthesis &amp;headers</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gpOneLiners">
         <property name="title">
          <string comment="@title:group">One Liners</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="chkKeepStatements">
            <property name="toolTip">
             <string comment="@info:tooltip">Do not break lines containing multiple statements into
multiple single-statement lines.</string>
            </property>
            <property name="text">
             <string comment="@option:check">&amp;Keep one-line statements</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkKeepBlocks">
            <property name="toolTip">
             <string comment="@info:tooltip">Do not break blocks residing completely on one line.</string>
            </property>
            <property name="text">
             <string comment="@option:check">Keep o&amp;ne-line blocks</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gpOther">
         <property name="title">
          <string comment="@title:group">Other</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
          <item row="0" column="0">
           <widget class="QLabel" name="lblPointerAlign">
            <property name="text">
             <string comment="@label:listbox">Pointer Alignment:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbPointerAlign">
            <item>
             <property name="text">
              <string comment="@item:inlistbox pointer alignment">No change</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string comment="@item:inlistbox pointer alignment">Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string comment="@item:inlistbox pointer alignment">Middle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string comment="@item:inlistbox pointer alignment">Type</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>KComboBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
